src/Providers/Lookup/FakeWasteLookupProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using form_builder.Models;
+using Newtonsoft.Json;
+using StockportGovUK.NetStandard.Gateways;
+
+namespace form_builder.Providers.Lookup
+{
+    public class FakeWasteLookupProvider : ILookupProvider
+    {
+        public string ProviderName { get => "FakeWaste"; }
```

Comment:
think there has been a bit of confusuion about the providers, There would not be multiple fake providers for different lookups, there would be different providers to support different response types so JSON or XML responses.